workers/data_refinery_workers/processors/smasher.py

```diff
@@ -541,16 +540,18 @@ def _update_result_objects(job_context: Dict) -> Dict:
 
 def smash(job_id: int, upload=True) -> None:
     """ Main Smasher interface """
-
     pipeline = Pipeline(name=PipelineEnum.SMASHER.value)
-    return utils.run_pipeline({ "job_id": job_id,
-                                "upload": upload,
-                                "pipeline": pipeline
-                            },
-                       [utils.start_job,
-                        smashing_utils.prepare_files,
-                        _smash_all,
-                        _upload,
-                        _notify,
-                        _update_result_objects,
-                        utils.end_job])
+    job_context = utils.run_pipeline({
+                                        "job_id": job_id,
+                                        "upload": upload,
+                                        "pipeline": pipeline
+                                     },
+                                     [utils.start_job,
+                                        smashing_utils.prepare_files,
+                                        _smash_all,
+                                        _upload,
+                                        _update_result_objects,
+                                        utils.end_job])
```

Comment:
I don't think this is an improvement to this formatting, but that's just like, my opinion.

Reply:
I do think it was messed up before though, it should have been lined up better.

Reply:
yes, not sure how to format this part. I'll note that I removed `_notify` from the pipelines methods and added it afterward so that it always gets executed. And we can throw failure exceptions while processing.

Reply:
fixed it!